asmcli/components/control-plane/in-cluster.sh

```diff
@@ -27,13 +27,13 @@ configure_in_cluster_control_plane() {
 
 init_meshconfig() {
   local USE_HUB_WIP; USE_HUB_WIP="$(context_get-option "USE_HUB_WIP")"
-  local HUB_MEMBERSHIP_ID; HUB_MEMBERSHIP_ID="$(context_get-option "HUB_MEMBERSHIP_ID")"
```

Comment:
HUB_MEMBERSHIP_ID in the context has not been set when cluster is registered with asmcli. Thus move this line after the populate_fleet_info.